src/app/fdctl/configure/workspace.c

```diff
@@ -14,7 +14,12 @@
 static void
 init_perm( fd_caps_ctx_t *  caps,
            config_t * const config ) {
-  ulong mlock_limit = fd_topo_mlock( &config->topo );
+  ulong mlock_limit = 0UL;
```

Comment:
Does this need to be a sum instead of a max if running in single process mode?

Reply:
Nope, these workspaces are created and then unmapped one by one.